### PR TITLE
Update the merlin preview for OCaml 5.0

### PR DESCRIPTION
**Pull request by kit-ty-kate**
cc @voodoos

### CODE REVIEW

**Comment by tmattio** [comment]
Thanks @kit-ty-kate @voodoos 🎉 
Shouldn't the version be 4.5~5.0.preview though? I understood the 500 branch was rebased on master

File: packages/merlin/merlin.4.4.1~5.0preview/opam [hunk 1]
```diff
@@ -13,14 +13,10 @@ flags: avoid-version
 depends: [
   "ocaml" {>= "5.0" & < "5.1"}
   "dune" {>= "2.9.0"}
+  "merlin-lib"
   "dot-merlin-reader" {>= "4.0"}
```
Comment (voodoos) [inline packages/merlin/merlin.4.4.1~5.0preview/opam:17]:
```suggestion
  "dot-merlin-reader" {>= "4.4~5.0.preview"}
```

File: packages/merlin/merlin.4.4.1~5.0preview/opam [hunk 2]
```diff
@@ -13,14 +13,10 @@ flags: avoid-version
...
-  "yojson" {>= "1.6.0" & < "2.0.0"}
+  "yojson" {>= "2.0.0"}
   "conf-jq" {with-test}
-  "csexp" {>= "1.2.3"}
-  "result" {>= "1.5"}
-  "menhir" {dev}
-  "menhirLib" {dev}
-  "menhirSdk" {dev}
 ]
```
Comment (kit-ty-kate) [inline packages/merlin/merlin.4.4.1~5.0preview/opam:20]:
```suggestion
]
conflicts: [
  "seq" {!= "base"}
]
```

**Comment by voodoos** [comment]
it gets weirder: 
> failed: Format errors detected in merlin.4.4.1~5.0preview: - At packages/merlin/merlin.4.4.1~5.0preview/opam:24:0-26:1:: Duplicate field conflicts

But I can see only one conflicts field...

**Pull request merged**